packages/components/doc/maintenance.md

```diff
@@ -0,0 +1,15 @@
+# Maintenance
+
+## Props
+
+|name|type|required|default|
+|---|---|---|---|---|
```

Comment:
You have 4 columns, not 5

Reply:
There is no description required really.